src/mqueryfront/src/query/QueryMatches.js

```diff
@@ -58,6 +92,12 @@ const QueryMatches = (props) => {
                                     itemList={downloadDropdownList}
                                 />
                             </span>
+                            {filters.length > 0 && (
+                                <span class="border rounded p-1 pull-right text-secondary">
```

Comment:
There's no `class` in React elements, use `className`
```suggestion
                                <span className="border rounded p-1 pull-right text-secondary">
```